packages/stv/src/index.test.ts

```diff
@@ -167,13 +167,13 @@ describe('calculateStvWinners', () => {
 
   it('should correctly eliminate the candidate with the fewest votes', () => {
     const voteRecords: VoteRecord[] = [
-      { voteCount: 2, voteOrder: ['Alice', 'Bob'] },
+      { voteCount: 1.5, voteOrder: ['Alice', 'Bob'] },
       { voteCount: 1, voteOrder: ['Charlie', 'Bob'] },
       { voteCount: 1, voteOrder: ['Bob', 'Charlie'] },
     ];
 
     const { winners, tieCount } = calculateStvWinners(voteRecords, 1);
-    expect(winners).toEqual(['Alice']); // Alice should win after Charlie is eliminated
+    expect(winners).toEqual(['Bob']); // Bob should win after Charlie is eliminated
```

Comment:
Updated the test so it would actually trigger an elimination (I assume it did originally, but not in the updated code).